packages/blocks/src/api/raw-handling/shortcode-converter.js

```diff
@@ -29,7 +29,13 @@ function segmentHTMLToShortcodeBlock( HTML, lastIndex = 0 ) {
 	}
 
 	const transformTags = castArray( transformation.tag );
-	const transformTag = first( transformTags );
+
+	let transformTag = first( transformTags );
+	for ( let i = 0; i < transformTags.length; i++ ) {
```

Comment:
Nice spotting!

For symmetry with the `findTransform` call's predicate at line 24, I would recommend [`find`](https://lodash.com/docs/4.17.15#find) instead of a `for` loop:

```js
const transformTag = find( transformTags, ( tag ) => regexp( tag ).test( HTML ) );
```